example/image_example_hdat.py

```diff
@@ -0,0 +1,55 @@
+from hdat import Suite, MetricsChecker
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .image_example import rotate_and_blur
+
+
+class ExampleSuite(Suite):
+    id = 'example-suite'
+
+    def collect(self):
+        return {
+            'chicken': {
+                'image_location': 'data/chicken.jpg',
+                'id': '1',
```

Comment:
Do we use the `id` at all?  If not, I suggest we remove it.

Reply:
I can't remember where exactly I was going with that. Maybe to show that you can have any number of things in the case data? But that's kind of obvious: I agree about removing it.